doc/sphinx/user_manual/publications.rst

```diff
@@ -86,7 +86,7 @@ Applications
 Course material
 ---------------
 
-.. [GM14duplicate] J.F. Groote, M.R. Mousavi. `Modeling and analysis of communicating systems <https://mitpress.mit.edu/books/modeling-and-analysis-communicating-systems>`__. The MIT press. 2014.
+.. [GM14b] J.F. Groote, M.R. Mousavi. `Modeling and analysis of communicating systems <https://mitpress.mit.edu/books/modeling-and-analysis-communicating-systems>`__. The MIT press. 2014.
```

Comment:
[GM14a] and [GM14b] are exactly the same book, wouldn't it be better to use [GM14] for both mentions? This also applies to `doc/sphinx/user_manual/tools/release/lps2lts.rst`

Reply:
Fixed in 7f98c77802bf44159255d772870a9d9aafcf7c03.